notebooks/Experimental/Ishan/ADP Demo/Data Owner - Canada.ipynb

```diff
@@ -1323,7 +1323,7 @@
    "source": [
     "# Let's convert the whole dataset to a private tensor\n",
     "\n",
-    "private_dataset_tensor = sy.Tensor(np_dataset).private(0.01, 1e15, entity=DataSubject(name=\"Canada\")).tag(\"private_canada_trade_dataset\")"
+    "private_dataset_tensor = sy.Tensor(np_dataset)annotate_with_dp_metadata(0.01, 1e15, entity=DataSubject(name=\"Canada\")).tag(\"private_canada_trade_dataset\")"
```

Comment:
NIT: I think we replaced annotate_with_dp_metadata without  `.`

Reply:
in the notebooks part.

Reply:
Fixed in: ca27bddcdb619027c8d8d7f3337e1bd6be7b59a2